lib/beaker-hostgenerator/data.rb

```diff
@@ -779,6 +779,22 @@ def osinfo
             'template' => 'ubuntu-1610-x86_64'
           }
         },
+        'vro6-64' => {
+          :general => {
+            'platform' => 'sles-11-x86_64'
+          },
+          :vmpooler => {
+            'template' => 'vro-6-x86_64'
+          }
+        },
+        'vro7-64' => {
+          :general => {
+            'platform' => 'sles-11-x86_64'
```

Comment:
It is in the puppet-vro-plugin rake file: https://github.com/puppetlabs/puppet-vro-plugin/blob/master/Rakefile#L148-L153